lib/autocomplete-provider.js

```diff
@@ -2,6 +2,8 @@
 
 import _ from 'lodash';
 
+import RobustGrammar from './robust-grammar';
```

Comment:
Since this isn't a class and is invoked as a function, use a leading lowercase like `robustGrammar`. In fact, I would probably call it `getRobustGrammar` or `getCurrentGrammar`.